projects/igniteui-angular/src/lib/accordion/README.md

```diff
@@ -0,0 +1,88 @@
+
+# IgxAccordion
+
+
+**IgxAccordion** is a container-based component that contains a collection of collapsible **IgxExpansionPanels**.
+
+A walkthrough of how to get started can be found [here](https://www.infragistics.com/products/ignite-ui-angular/angular/components/accordion)
+
```

Comment:
Might be a good idea to include a `Dependencies` section somewhere in the beginning since one of the first things a user will want to see is how to actually set up the accordion.

Reply:
I feel a `Dependencies` section would be redundant, as it would either need to state that you have to import the appropriate module (`IgxAccordionModule`) in your project (which is pretty self-explanatory, is featured in the docs and is fundamental for any component) or that you have to import `BrowserAnimationsModule` in your app root (which is stated in the `getting started` section and is already part of our `ng add` migration).
I'll agree that there are some inconsistency in these sections and a `Dependency` section is not features in most of the other components' READMEs, but I think it's not necessary to add one.